linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/ansible/shared.yml

```diff
@@ -4,12 +4,7 @@
 # complexity = low
 # disruption = medium
 
-- name: Ensure NetworkManager is installed
-  ansible.builtin.package:
-    name: "{{ item }}"
-    state: present
-  with_items:
-    - NetworkManager
+{{{ ansible_package_install(["NetworkManager"]) }}}
```

Comment:
Attention at the task below this. The task uses "when:" statement where the package name will have to be overridden as well